### PR TITLE
[bitnami/kong] Update test of goss

### PR DESCRIPTION
**Pull request by Mauraza**
### Description of the change

Update the test of Kong for the new version.

### Applicable issues

- fixes #

### CODE REVIEW

File: .vib/kong/goss/vars.yaml [hunk 1]
```diff
@@ -8,10 +8,6 @@ directories:
       - /opt/bitnami/kong/conf
   - paths:
       - /docker-entrypoint-initdb.d
-      - /opt/bitnami/kong/openresty/luajit/share/lua/5.1/kong/include/opentelemetry
+      - /opt/bitnami/kong/include/opentelemetry
```
Comment (FraPazGal) [inline .vib/kong/goss/vars.yaml:11]: We are already checking this path is a symlink in `kong.yaml`, I don't think we need to check this here.

File: .vib/kong/goss/vars.yaml [hunk 2]
```diff
@@ -8,10 +8,6 @@ directories:
...
       - /opt/bitnami/kong/bin
-      - /opt/bitnami/kong/server/lib
-files:
-  - paths:
-      - /opt/bitnami/kong/server/lib/pluginsocket.proto
```
Comment (FraPazGal) [inline .vib/kong/goss/vars.yaml:16]: Is the `../kong/server/lib` folder no longer present? Is this related to the Lua deprecation as well?

**Review by FraPazGal — COMMENTED** [review]
Hi @Mauraza, I added a couple of comments. Please check them whenever you can

File: .vib/kong/goss/vars.yaml (hunk 2 above)
Comment (Mauraza) [reply .vib/kong/goss/vars.yaml:16]: It does not exist anymore:
```bash
root@04b2c80004a5:/# ls /opt/bitnami/kong/server/
root@04b2c80004a5:/# 
```

**Review by FraPazGal — APPROVED** [review]
LGTM!
